src/db_ledger.rs

```diff
@@ -329,6 +331,7 @@ impl DbLedger {
     }
 
     pub fn destroy(ledger_path: &str) -> Result<()> {
+        create_dir_all(&ledger_path)?;
```

Comment:
this is weird...

Reply:
yeah it deserves a comment, added.  DB::Destroy fails if the parent directory doesn't exist 🙃